index.js

```diff
@@ -47,25 +21,29 @@ closeNavButton.addEventListener("click", () => {
 });
 
 addTaskButton.addEventListener("click", () => {
-  tasks.push({
-    id: tasks.length + 1,
-    title: taskTitleInput.value,
-    isDone: false,
-    createdAt: new Date().toISOString(),
-  });
-  console.log(tasks);
-  renderTasks(tasks);
+  if (Number(taskTitleInput.value) == 0) {
+    alert("Task title input can not be empty!");
```

Comment:
Show this message under the input with red color instead of using alert.

Reply:
Got it